ui/lib/liquidity-rewards.ts

```diff
@@ -68,12 +68,14 @@ export function useLiquidityRewards({ nationPrice, poolValue, address }: any) {
 
   useEffect(() => {
     if (totalRewards && poolValue && totalDeposit && lpTokensSupply) {
-      setLiquidityRewardsAPY(
-        totalRewards
-          .mul(transformNumber(12 / months, NumberType.bignumber))
-          .mul(transformNumber(nationPrice, NumberType.bignumber, 2))
-          .div(poolValue.mul(totalDeposit).div(lpTokensSupply))
-      )
+      if (totalDeposit.gt(transformNumber(0, NumberType.bignumber))){
```

Comment:
@brymut `totalDeposit > 0` does not work here?

Reply:
Since this is a big Number type, thought this would be more appropriate